app/models/vm/operations.rb

```diff
@@ -18,6 +18,10 @@ module Vm::Operations
       unsupported_reason_add(:vmrc_console, _("VMRC Console not supported")) unless console_supported?('VMRC')
     end
 
+    supports :native_console do
+      unsupported_reason_add(:native_console, _("NATIVE Console not supported")) unless console_supported?('NATIVE')
```

Comment:
Should this say "VM NATIVE Console not supported" to be consistent with your line 40 in this file?

Reply:
Yes, I'll change it like the one in line 20